src/common/classes/fb_string.h

```diff
@@ -411,10 +411,10 @@ namespace Firebird
 		}
 		void insert(size_type p0, const_pointer s, const size_type n)
 		{
```

Comment:
No, this is regression of this branch, problem does not exist in other branches.